Ptt/API/APIClient.swift

```diff
@@ -201,9 +201,11 @@ extension APIClient: APIClientProtocol {
             urlComponent = goPttBBSURLComponents
             urlComponent.path = "/api/board/\(bid)/articles"
             // Percent encoding is automatically done with RFC 3986
-            urlComponent.queryItems = [
-                URLQueryItem(name: "start_idx", value: startIdx)
-            ]
+            if let startIdx {
+                urlComponent.queryItems = [
+                    URLQueryItem(name: "start_idx", value: startIdx)
+                ]
+            }
         case .go_bbs(boardID: let boardID):
```

Comment:
out of curios, do we have backend approach decision?

Reply:
Not yet. We’re currently focusing on implementing with go_pttbbs, but still open to other solutions.